scripts/metavar.py

```diff
@@ -85,6 +85,12 @@
                           'horizontal_loop_begin', 'horizontal_loop_end',
                           'vertical_layer_index', 'vertical_interface_index']
 
+# DH* Is there a better place for these definitions?
```

Comment:
The metavar class needs this in order to parse the variable active attribute and turn it into a Fortran conditional. This is copied straight from ccpp_prebuild.py where it's been in use (i.e. tested extensively) in the UFS, NEPTUNE and SCM.

Reply:
Move this to some place in Fortran subdirectory